### ISSUE TITLE
BDI controller for old atlas model broken on default

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#432]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/432)) by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
Since [pull request #447](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/447) (atlas shim interface), the BDI controller doesn't work with the old atlas model.

I can't make atlas walk with the following launch files. It would be nice to retain the old BDI controller for these old models so VRC controllers will still work.
~~~
roslaunch drcsim_gazebo atlas.launch
roslaunch drcsim_gazebo atlas_sandia_hands.launch
~~~

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
I've updated [pull request #451](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/451) so it should retain the old plugin. Please give it a try. Thanks.

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
It looks like this will be resolved by [pull request #451](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/451).

**Comment by osrf-migration** [comment]
**Original comment by Nate Koenig (Bitbucket: [Nathan Koenig](https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/)).**

----------------------------------------

* changed state from "new" to "resolved"

Resolved in [pull request #451](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/451)
